ocrd_network/ocrd_network/deployment_utils.py

```diff
@@ -101,7 +103,16 @@ def wait_for_rabbitmq_availability(
         else:
             # TODO: Disconnect the dummy_publisher here before returning...
             return
-    raise RuntimeError('Error waiting for queue startup: timeout exceeded')
+    raise RuntimeError(f'Cannot connet to Rabbitmq host: {host}, port: {port}, '
+                       f'vhost: {vhost}, username: {username}')
+
+
+def verify_mongodb_available(mongo_url: str) -> None:
+    try:
+        client = MongoClient(mongo_url, serverSelectionTimeoutMS=1000.0)
+        client.admin.command("ismaster")
+    except Exception:
+        raise RuntimeError(f'Cannot connet to MongoDb: {re.sub(r":[^@]+@", ":****@", mongo_url)}')
```

Comment:
Typo: connet -> connect. Also, `MongoDB` (in other places as well).

Reply:
Thanks, fixed as well